pkgs/cronet_http/tool/run_pigeon.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/sh
 
 # Generate the platform messages used by cronet_http.
+cd ../
```

Comment:
Why did you change this?

Reply:
It's under the root path previously, now it's under `/tool`.

Reply:
OK.